src/chat/takeover.js

```diff
@@ -359,4 +359,12 @@ var takeover = module.exports = function() {
             }
         }
     });
+
+    $('.tse-content').on('click', '.chat-line', function() {
```

Comment:
Not specific enough. Bind to names themselves. If someone wants to bind to the chat message itself (like say we want to bring back double-click to translate), it would call this listener.

I would also recommend doing a double-click to insert. If someone wants to right-click, they shouldn't be forced to perform a different action.